configure.py

```diff
@@ -1911,7 +1911,9 @@ def insert_join(match):
                         else:
                             output += for_body.replace('%{i}', v).replace('%{i|upper}', v.upper())
 
-                        omitlast_match = self.omitlast_pattern.match(output)
+                        omitlast_match = None
+                        if output.find('%{omitlast') >= 0:
+                            omitlast_match = self.omitlast_pattern.match(output)
                         if omitlast_match:
                             output = omitlast_match.group(1)
```

Comment:
Nit: perhaps like that, to please the python gods and avoid pre-defining the `omitlast_match` variable?

```python
if '%{omitlast' in output:
    if omitlast_match := self.omitlast_pattern.match(output):
        output = omitlast_match.group(1)
        if i + 1 < len(var):
            output += omitlast_match.group(2)
        output += omitlast_match.group(3)
```

Reply:
I tried with a ternary like: `match = ol_pattern.match() if "omitlast" in output else None`, but that does seem to execute the regex regardless of the `if` statement. :(